administration/troubleshooting.md

```diff
@@ -140,6 +140,50 @@ Now we should start seeing output similar to the following:
+* Copyright (C) 2015-2022 The Fluent Bit Authors
+* Fluent Bit is a CNCF sub-project under the umbrella of Fluentd
+* https://fluentbit.io
+
+[2023/01/27 07:44:25] [ info] [fluent bit] version=2.0.8, commit=9444fdc5ee, pid=1
+[2023/01/27 07:44:25] [ info] [storage] ver=1.4.0, type=memory, sync=normal, checksum=off, max_chunks_up=128
+[2023/01/27 07:44:25] [ info] [cmetrics] version=0.5.8
+[2023/01/27 07:44:25] [ info] [ctraces ] version=0.2.7
+[2023/01/27 07:44:25] [ info] [input:dummy:input_dummy] initializing
+[2023/01/27 07:44:25] [ info] [input:dummy:input_dummy] storage_strategy='memory' (memory only)
+[2023/01/27 07:44:25] [ info] [output:stdout:stdout.0] worker #0 started
+[2023/01/27 07:44:25] [ info] [http_server] listen iface=0.0.0.0 tcp_port=2020
+[2023/01/27 07:44:25] [ info] [sp] stream processor started
+[0] dummy.0: [1674805465.976012761, {"message"=>"dummy"}]
+[0] dummy.0: [1674805466.973669512, {"message"=>"dummy"}]
+...
+```
+Next, in another terminal, we activate Tap including the output, in this case stdout, and the parameters wanted, in this case ```"format": "json"```:
+
+```shell
+$ curl 127.0.0.1:2020/api/v1/trace/input_dummy -d '{"output":"stdout", "params": {"format": "json"}}'
+{"status":"ok"}
+```
+In the first terminal, we should be seeing the output similar to the following:
+```shell
+[0] dummy.0: [1674805635.972373840, {"message"=>"dummy"}]
+[{"date":1674805634.974457,"type":1,"trace_id":"0","plugin_instance":"dummy.0","plugin_alias":"input_dummy","records":[{"timestamp":1674805634,"record":{"message":"dummy"}}],"start_time":1674805634,"end_time":1674805634},{"date":1674805634.974605,"type":3,"trace_id":"0","plugin_instance":"dummy.0","plugin_alias":"input_dummy","records":[{"timestamp":1674805634,"record":{"message":"dummy"}}],"start_time":1674805634,"end_time":1674805634},{"date":1674805635.972398,"type":1,"trace_id":"1","plugin_instance":"dummy.0","plugin_alias":"input_dummy","records":[{"timestamp":1674805635,"record":{"message":"dummy"}}],"start_time":1674805635,"end_time":1674805635},{"date":1674805635.972413,"type":3,"trace_id":"1","plugin_instance":"dummy.0","plugin_alias":"input_dummy","records":[{"timestamp":1674805635,"record":{"message":"dummy"}}],"start_time":1674805635,"end_time":1674805635}]
+[0] dummy.0: [1674805636.973970215, {"message"=>"dummy"}]
+[{"date":1674805636.974008,"type":1,"trace_id":"2","plugin_instance":"dummy.0","plugin_alias":"input_dummy","records":[{"timestamp":1674805636,"record":{"message":"dummy"}}],"start_time":1674805636,"end_time":1674805636},{"date":1674805636.974034,"type":3,"trace_id":"2","plugin_instance":"dummy.0","plugin_alias":"input_dummy","records":[{"timestamp":1674805636,"record":{"message":"dummy"}}],"start_time":1674805636,"end_time":1674805636}]
+```
+This parameter shows stdout in Json format, however, as mentioned before, parameters can be passed to any plugin. 
+
+Please visit the following link for more information on other output parameters:
```

Comment:
output plugins, instead of output parameters.

Reply:
done